### PR TITLE
Package multiple python modules

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change

I want to package these modules so that I can use a nix-shell for development.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by ivegotasthma** [comment]
@dotlambda ping for done

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1943,6 +1943,15 @@
...
     name = "Ivan Tkatchev";
   };
+  ivegotasthma = {
+    email = "ivegotasthma@protonmail.com";
+    github = "ivegotasthma";
+    name = "John Doe";
+    keys = [{
+      longkeyid = "rsa4096/09AC52AEA87817A4";
+      fingerprint = "4008 2A5B 56A4 79B9 83CB  95FD 09AC 52AE A878 17A4";
+    }];
```
Comment (dotlambda) [inline maintainers/maintainer-list.nix]: Where is this format for `maintainer-list.nix` documented?

Comment (ivegotasthma) [reply maintainers/maintainer-list.nix]: https://github.com/NixOS/nixpkgs/pull/47663

**Comment by dotlambda** [comment]
@GrahamcOfBorg build pythonPackages.djangorestframework-jwt python3Packages.djangorestframework-jwt pythonPackages.django-rest-auth python3Packages.django-rest-auth pythonPackages.django-cors-headers python3Packages.django-cors-headers pythonPackages.braintree python3Packages.braintree

**Comment by ivegotasthma** [comment]
@dotlambda thanks!

**Pull request merged**